OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -25,27 +25,39 @@
     <Reference Include="Eluant, Version=1.0.5229.27703, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\thirdparty\Eluant.dll</HintPath>
-      <Private>False</Private>
+      <Private>True</Private>
```

Comment:
Referencing private copies interacts badly with our runtime assembly loading, and can lead to multiple distinct copies of the same classes being loaded.  This leads to difficult to debug crashes such as #3038 or #6711.

I'm hoping that somebody with direct experience with these windows project issues will be able to jump in with a bright idea on how to proceed.